shared/team-building/container.tsx

```diff
@@ -680,6 +682,10 @@ const mergeProps = (
         : ownProps.decHighlightIndex,
     recommendations: recommendationsSections,
     recommendedHideYourself: ownProps.recommendedHideYourself,
+    refreshBlockList: () =>
+      ownProps.namespace === 'people' && searchResults
+        ? dispatchProps._refreshBlockList(searchResults.map(r => r.services['keybase'] || '').filter(r => r))
```

Comment:
could do `searchResults.map(r=>r.services['keybase']).filter(Boolean)` so it's more clear you are filtering out empty ones.

Reply:
Agree here